packages/pluggableWidgets/switch-web/src/components/Switch.tsx

```diff
@@ -0,0 +1,46 @@
+import { MouseEvent, KeyboardEvent, createElement } from "react";
+import classNames from "classnames";
+import { Alert } from "@mendix/piw-utils-internal";
+import { SwitchContainerProps } from "../../typings/SwitchProps";
+
+export interface SwitchProps extends Pick<SwitchContainerProps, "id" | "class" | "style" | "tabIndex" | "deviceStyle"> {
+    onClick?: (event: MouseEvent<HTMLDivElement>) => void;
+    onKeyDown?: (event: KeyboardEvent<HTMLDivElement>) => void;
+    isChecked: boolean;
+    editable: boolean;
+    validation: string | undefined;
+}
+
+export default function Switch(props: SwitchProps) {
+    return (
+        <div className={classNames("widget-switch", props.class, props.deviceStyle)} style={props.style}>
+            <input
+                checked={props.isChecked}
+                className={classNames("widget-switch-checkbox", { enabled: props.editable })}
+                readOnly
+                type="checkbox"
+            />
+            <div
+                className={classNames("widget-switch-btn-wrapper", "widget-switch-btn-wrapper-default", {
+                    checked: props.isChecked,
+                    disabled: !props.editable,
+                    "un-checked": !props.isChecked
+                })}
+                onClick={props.onClick}
+                onKeyDown={props.onKeyDown}
+                tabIndex={props.tabIndex ?? 0}
+                role="checkbox"
+                aria-checked={props.isChecked}
+                aria-labelledby={props.id}
+            >
+                <small
```

Comment:
This comes from Switch 2.0.0. I don't think changing it outweighs backwards compatibility.

Reply:
I think this decision depends on the Mendix version that this widget is going to support. If it will be only mx 9, it doesn't hurt to do breaking DOM changes. Only migration of customer styling will be a pain.

Reply:
From Chris, it is going to be compatible with MX9 and Atlas 3. An outstanding discussion remains whether the styling in Atlas (for this widget) should support widget's similar to Switch 2.0.0 structure. I think it is a good thing to do.

Reply:
We want to keep DOM changes to a minimal, but if this HTML element doesn't describe the purpose of what it contains we should change it.

Reply:
> An outstanding discussion remains whether the styling in Atlas (for this widget) should support widget's similar to Switch 2.0.0 structure.
Are the compatible styles you included enough for this?

Reply:
RE changing `small`, I think it was discussed while I was away that it's fine. See [this](https://github.com/mendix/widgets-resources/pull/559#discussion_r611388596) comment.

Reply:
> Are the compatible styles you included enough for this?

Yes

Reply:
Did you include all Atlas 2 styles for switch 2.x in Atlas 3?

Reply:
Yes

Reply:
I changed `small` => `div`

Reply:
@Keraito if you can please review for Isa